elections/tac-and-scir-election-process.md

```diff
@@ -73,7 +73,15 @@ The Governing Board will appoint three people from the list of TAC and SCIR Nomi
 
 ## TAC Chair and Vice Chair Election
 
-After the results are announced the TAC members will elect a Chair and a Vice Chair with 1 week for nominations and 1 week for votes. Results may be announced when all votes are cast. The candidate with the most votes will serve as Chair and the second as Vice Chair. The Chair may however choose to swap seat with the Vice Chair if agreed by the Vice Chair.
+After the results of the community elections and Governing Board appointments are announced, the TAC members will elect a [Chair and a Vice Chair](/process/tac-member-R&Rs.md#tac-chair).
+
+The TAC Chair nomination period shall commence as soon as new TAC members are seated (typically, at the beginning of the year) and shall conclude at the end of the first TAC meeting of the year.
+
+OpenSSF staff will coordinate the election process and results will be announced soon after all votes have been cast.
```

Comment:
I think we should add (back) a bit more detail and a concrete timeline to this aspect of the process. For instance, stating that the election period is 1 week, starting after the conclusion of the nomination phase / the end of the first TAC meeting. This adds clarity to the process (helps staff) and is in fact in alignment with the currently documented process and what we decided to do this year.

Reply:
In line with my comment above, I suggest to add a concrete timeline for the voting period. The suggestion here is 1 week, as it is also currently documented.

```suggestion
The voting period is 1 week, starting immediately after the conclusion of the nomination period. OpenSSF staff will coordinate the election process and results will be announced soon after all votes have been cast.
```